src/helpers/getPDFCoverPage.ts

```diff
@@ -0,0 +1,65 @@
+};
+
+export const PDF_COVER_PAGE_WIDTH = 595;
+export const PDF_COVER_PAGE_HEIGHT = 842;
+
+const getPDFCoverPage = ({
+  certificateTitle = '',
+  issueDate = '',
+  issuerName = '',
+  recipientName = '',
+  issuerPublicKey = '',
+  issuerLogo = '',
+  qrCodeImage = ''
+}: {
+  certificateTitle?: string;
+  issueDate: string;
+  recipientName: string;
+  issuerName: string;
+  issuerPublicKey: string;
+  issuerLogo?: string;
+  qrCodeImage?: string;
+}): HTMLElement => {
+  const html: string[] = [];
+
+  if (certificateTitle || issuerLogo) {
+    html.push('<section>');
+
+    if (issuerLogo) {
+      html.push(`<div style="margin:0 auto;width:120px;height:120px;background-repeat:no-repeat;background-position:center;background-size:contain;background-image:url('${issuerLogo}');"></div>`);
+    }
+
+    if (certificateTitle) {
+      html.push(`<h2 style="margin:30px 0 20px">${certificateTitle}</h2>`);
+    }
+
+    html.push('</section>');
```

Comment:
in webcomponent world, this is a bit dirty

Reply:
I know, not great but I do need the HTML stringified, and if I invoke `TemplateResult.getHtml()` I don't get the inner logic (i18n, sub-functions,...)